src/chainstate/stacks/auth.rs

```diff
@@ -1913,20 +2238,93 @@ mod test {
             0x00,
             0x00,
             0x00,
-            0x01,
-            0x59,
+            0x00,
+            0x7b,
             // fee rate
             0x00,
             0x00,
             0x00,
             0x00,
             0x00,
             0x00,
-            0x02,
-            0x37,
-            // key encoding
-            TransactionPublicKeyEncoding::Compressed as u8,
-            // signature
+            0x01,
```

Comment:
So, I think an older version of `cargo fmt` forced each of these array elements to be on one line.  It's fine with me (encouraged, even) to put all array elements for one serialized field on the same line, or in a 128-ish column block, to improve legibility :)